src/status_im/utils/js_resources.cljs

```diff
@@ -48,4 +48,10 @@
 (defn web3-init [provider-address]
   (str "var providerAddress = \"" provider-address "\";"
        (slurp "resources/web3_init.js")))
- ;;
\ No newline at end of file
+
+(defn local-storage-data [data]
+  (str "var localStorageData = "
+       (if data
+         data
+         "{}")
```

Comment:
Default for localStorage#getItem is `null`. Do we want to emulate this? It would help distinguish non-existent keys.